subsys/bluetooth/controller/Kconfig

```diff
@@ -7,6 +7,56 @@
 
 comment "BLE Controller support"
 
+# The following symbols are enabled depending if the controller actually
+# supports the respective features. In case of Nordic SW LL, all of these are
+# supported; for OpenISA SW LL, they're not
+
+config BT_LL_SW_LEGACY_SUPPORT
+	bool
+	depends on SOC_COMPATIBLE_NRF
+
+config BT_PHY_UPDATE_SUPPORT
+	bool
+
+config BT_DATA_LEN_UPDATE_SUPPORT
+	bool
+
+config BT_HAS_HCI_VS_SUPPORT
+	bool
+
+config BT_CTLR_LE_ENC_SUPPORT
+	bool
+
+config BT_CTLR_CONN_PARAM_REQ_SUPPORT
+	bool
+
+config BT_CTLR_EXT_REJ_IND_SUPPORT
+	bool
+
+config BT_CTLR_SLAVE_FEAT_REQ_SUPPORT
+	bool
+
+config BT_CTLR_PRIVACY_SUPPORT
+	bool
+
+config BT_CTLR_EXT_SCAN_FP_SUPPORT
+	bool
+
+config BT_CTLR_CHAN_SEL_2_SUPPORT
+	bool
+
+config BT_CTLR_ADV_EXT_SUPPORT
+	bool
+
+config BT_CTLR_XTAL_ADVANCED_SUPPORT
```

Comment:
@cvinayak These last 3 seems very nordic specific, should these perhaps be moved under LLL_VENDOR_NORDIC?

Reply:
Agree.

Reply:
I can address them as part of a PR if @cvinayak agrees.